srcpkgs/factor/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'factor'
+pkgname=factor
+version=0.97
+revision=1
+only_for_archs="x86_64"
+build_style=gnu-makefile
+hostmakedepends="unzip pkg-config"
+makedepends="gtkglext-devel"
+nostrip_files="a.elf"
+short_desc="Factor: a concatenative programming language, similar to Forth"
+maintainer="B. Wilson <x@wilsonb.com>"
+license="BSD"
+homepage="http://factorcode.org/"
+distfiles="http://downloads.factorcode.org/releases/${version}/factor-src-${version}.zip"
```

Comment:
https if possible

Reply:
Yeah. It looks like their cert is invalid. Currently trying to work with upstream to get it fixed.